pages/listings/listing/create/index.html

```diff
@@ -136,8 +136,8 @@ <h1 class="text-center mt-4">Create New Listing</h1>
           <div class="modal-dialog">
             <div class="modal-content text-center">
               <div class="modal-body">
-                <!-- <img src="/public/assets/icons/cancel.svg" alt="" />
-                <h3>Woops! Please try again.</h3> -->
+                <img src="/assets/icons/cancel.svg" alt="Cancle icon" />
```

Comment:
There's a typo in your alt=""
Is alt="Cancle icon"
Should be alt="Cancel icon"

Reply:
fixed